gaeul/source/source-application.c

```diff
@@ -400,6 +403,10 @@ gaeul_source_application_command_line (GApplication * app,
     passphrase = g_settings_get_string (ssettings, "passphrase");
     pbkeylen = g_settings_get_enum (ssettings, "pbkeylen");
 
+    if (prefer_hw_decoding) {
+      g_object_set (pipeline, "prefer-hw-decoding", TRUE, NULL);
+    }
+
```

Comment:
Since `prefer-hw-decoding` is boolean, I think it can simply set directly from settings.

```
  g_object_set (pipeline, "prefer-hw-decoding", g_settings_get_boolean (ssetings, "prefer-hw-decoding"), NULL);
```

Reply:
Modified as per the suggestion